python_modules/libraries/dagster-elt/dagster_elt/asset_defs.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Iterable, Optional, Union
+from dagster import AssetsDefinition, asset, AssetKey, AssetExecutionContext
+from dagster._core.definitions.events import CoercibleToAssetKey
+from . import SlingMode, SlingResource, SlingSource, SlingTarget
+import re
+
+
+def build_sling_asset(
+    key: AssetKey,
```

Comment:
I would call this `asset_key` to disambiguate it from any key concept inside sling.